examples/sql_bank/main.go

```diff
@@ -142,25 +155,31 @@ func verifyBank(db *sql.DB) {
 }
 
 func main() {
-	db, err := sql.Open("postgres", "dbname=postgres sslmode=disable")
+	security.SetReadFileFn(securitytest.Asset)
+	serv := server.StartTestServer(nil)
+	defer serv.Stop()
+
+	url := "https://root@" + serv.ServingAddr() + "?certs=test_certs"
```

Comment:
can we preserve the ability to run this on postgres? a command line flag or environment variable would do.